.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 18.14.0
+yarn 1.22.19
```

Comment:
there's no need to include the .tool-version , you can ignore it

Reply:
that one slipped, deleting